settings.gradle

```diff
@@ -12,7 +12,8 @@ dependencyResolutionManagement {
       library('guava', 'com.google.guava:guava:31.1-jre')
       library('gson', 'com.google.code.gson:gson:2.9.0')
       library('jsr305', 'com.google.code.findbugs:jsr305:3.0.2')
-      library('servlet-api', 'javax.servlet:javax.servlet-api:3.1.0')
+      library('javax-servlet-api', 'javax.servlet:javax.servlet-api:3.1.0')
+      library('jakarta-servlet-api', 'jakarta.servlet:jakarta.servlet-api:6.0.0')
```

Comment:
I would drop jakarta implementation for now. I have no idea how Tomcat will behave if we will provide both libraries.

Reply:
hm, okay - make sense